paddle/fluid/eager/grad_tensor_holder.cc

```diff
@@ -130,6 +131,26 @@ void GradTensorHolder::add(size_t slot_id, size_t rank,
                                                   &new_buffer);
         buffer_tensor.set_impl(new_buffer.impl());
       }
+    } else if (t.is_sparse_coo_tensor()) {
```

Comment:
support this is accumulation_node.cc

Reply:
and how about dense add to sparse?

Reply:
> support this is accumulation_node.cc

done

> and how about dense add to sparse?

The sparse API requires that the gradient of SparseTensor is still a SparseTenosr, so dense+sparse is not required at present.